add_to_pydotorg.py

```diff
@@ -340,14 +340,36 @@ def has_sigstore_signature(filename: str) -> bool:
 
     if unsigned_files:
         print("Signing release files with Sigstore")
-        run_cmd(
-            ["python3", "-m", "sigstore", "sign", "--oidc-disable-ambient-providers"]
-            + unsigned_files
-        )
-        for file in unsigned_files:
-            run_cmd(["chmod", "644", file + ".sig"])
-            run_cmd(["chmod", "644", file + ".crt"])
-            run_cmd(["chmod", "644", file + ".sigstore"])
+        for filename in unsigned_files:
+            cert_file = filename + ".crt"
+            sig_file = filename + ".sig"
+            bundle_file = filename + ".sigstore"
+
+            run_cmd(
+                [
+                    "python3",
+                    "-m",
+                    "sigstore",
+                    "sign",
+                    "--oidc-disable-ambient-providers",
+                    "--signature",
+                    sig_file,
+                    "--certificate",
+                    cert_file,
+                    "--bundle",
+                    bundle_file,
+                    filename,
+                ]
+            )
+
+            # New sigstore CLI makes Sigstore bundles end in '.json'
+            # but our users expect bundles to be '.sigstore'
+            if os.path.exists(bundle_file + ".json"):
+                os.rename(bundle_file + ".json", bundle_file)
```

Comment:
I might be wrong but this should never happen, right? If you're explicitly passing `--bundle {bundle_file}` above, then sigstore-python should never emit the newer `.sigstore.json` suffix, only whatever filename you explicitly pass 🙂

Reply:
Done in https://github.com/python/release-tools/pull/159/commits/4f724be99dd83c58f8dd0ad63956ca912c7de8c3